docs/source/run/plugin.rst

```diff
@@ -0,0 +1,132 @@
+================
+Managing plugins
+================
+
+Sopel is designed with plugins in mind: they add features, commands, and can
+do many things to make Sopel more useful or more fun to interact with on IRC.
+
+They are two mains type of plugins you can add:
+
+* single file plugins that you drop into a folder
+* packaged plugins that you install with ``pip``
+
+A plugin can have its own configuration section, and may generate files or use
+the database.
+
+.. note::
+
+    Make sure to read the documentation for the plugins you install and use.
+
+
+Adding plugins
+==============
+
+Install single file plugins
+---------------------------
+
+To add a single file plugin to your Sopel instance, you need to copy the file
+into its ``plugins`` directory. Said directory is relative to the
+:attr:`~sopel.config.core_section.CoreSection.homedir` directory, which means
+by default the plugin file should be copied into ``~/.sopel/plugins``.
+
+.. note::
+
+    Following the :doc:`systemd service example <service>`, if your homedir is
+    ``/var/lib/sopel`` then your plugin directory is
+    ``/var/lib/sopel/plugins``.
+
+If there is no configuration to change for the plugin, you can directly start
+(or restart) your bot and Sopel will load it automatically.
+
+Install packaged plugins
+------------------------
+
+Plugin authors might also publish their works as packages; you can find them by
+searching PyPI, or by using your favorite search engine to search for
+e.g. “sopel reminder plugin”.
```

Comment:
```suggestion
e.g. "sopel reminder plugin".
```

Smart-ass text editor you've got there. 🙃

Reply:
Probably a copy/paste of something else. I don't use/have smart editor replacement for quotes.